docs/how-to/test_upf_performance.md

```diff
@@ -0,0 +1,141 @@
+# Test UPF Performance
+
+This guide covers how to test the performance of the UPF.
+
+## Requirements
+
+You will need for this test a working deployment of Charmed Aether SD-Core by following the [Accelerated Networking Tutorial](../tutorials/accelerated_networking.md).
```

Comment:
Is this really a pre-requisite? We should probably be able to assess the upf throughput in standalone deployments.

Reply:
By standalone, do you mean the UPF running without a core, or one of the other tutorials?

I did not test this with the `Getting Started` tutorial, but I initially targeted the `Mastering` tutorial, and it did not work with that setup, because of TCP checksum issues that I was not able to fix.

For the `Getting Started` tutorial, because everything runs in the same K8s cluster, there would be no VM available where to install the `ueransim` snap. We could package it for Kubernetes, but it would be more involved. We could also investigate running it on the host, but we would need to make sure it gets network access to the AMF and to the UPF.

Reply:
How about creating a how-to which is not based on any tutorial? We could set up a dedicated env using Terraform, reusing relevant parts of the module you wrote for the setting up infra for the Mastering tutorial. Would that make sense? 

Right now my feeling is that the number of layers (this how-to customizes the Accelerated Networking tutorial, which customizes the Mastering tutorial) you need to go through to get this running can be confusing.

Reply:
Since I got the approval, I will merge this as-is, with the understanding that this is a first version and we need to make improvements to our documentation in general.